ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/Topic.java

```diff
@@ -92,7 +93,7 @@ Serializer getValueSerializer(final SchemaRegistryClient schemaRegistryClient) {
 
   Deserializer getValueDeserializer(final SchemaRegistryClient schemaRegistryClient) {
     final Deserializer<?> deserializer = valueSerdeSupplier.getDeserializer(schemaRegistryClient);
-    deserializer.configure(ImmutableMap.of(), false);
+    deserializer.configure(ImmutableMap.of("schema.registry.url", "foo"), false);
```

Comment:
why do we need this?

Reply:
`AbstractConfig` in `AbstractKafkaAvroSerDeConfig` now requires the schema registry URL be set, otherwise, it complains with the following exception:
`io.confluent.common.config.ConfigException: Missing required configuration "schema.registry.url" which has no default value.`